.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install latest nightly
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2020-09-20
+          toolchain: nightly
```

Comment:
is required? or only test use `nightly`, if is must change to `nightly` .github/workflows/release.yml need change follow this.
and this file change may conflict pr #632

Reply:
Not sure the minimal support version.

But 09-20 not works.

So I set to latest nightly directly.

Reply:
I think there can be add a specific nightly version, this way will help us to generate WASM have same proposal hash; and may need test `nightly-2020-10-27` is support this pr. because paritytech/srtool only have `nightly-2020-10-27`. if not support we need change use https://gitlab.com/chevdor/srtool ,  so this pr can merge first, when merged, I'll solve conflict and change CI rust toolchain version. and then build a new tag. currently need to find a suitable version.

Reply:
Let's talk this while 0.8.30 coming.